packages/api/tests/endpoints/test-executions.js

```diff
@@ -147,21 +179,86 @@ test('executions can be filtered by workflow', async (t) => {
 });
 
 test('GET returns an existing execution', async (t) => {
-  const response = await request(app)
-    .get(`/executions/${fakeExecutions[0].arn}`)
+  const collectionRecord = fakeCollectionRecordFactory();
+  const asyncRecord = fakeAsyncOperationRecordFactory();
+  const parentExecutionRecord = fakeExecutionRecordFactory();
+
+  const executionPgModel = new ExecutionPgModel();
+  const collectionPgModel = new CollectionPgModel();
+  const asyncOperationsPgModel = new AsyncOperationPgModel();
+
+  const [collectionCumulusId] = await collectionPgModel.create(
+    t.context.knex,
+    collectionRecord
+  );
+
+  const [asyncOperationCumulusId] = await asyncOperationsPgModel.create(
+    t.context.knex,
+    asyncRecord
+  );
+
+  const [parentExecutionCumulusId] = await executionPgModel.create(
+    t.context.knex,
+    parentExecutionRecord
+  );
+
+  const executionRecord = await fakeExecutionRecordFactory({
+    async_operation_cumulus_id: asyncOperationCumulusId,
+    collection_cumulus_id: collectionCumulusId,
+    parent_cumulus_id: parentExecutionCumulusId,
+  });
+
+  await executionPgModel.create(
+    t.context.knex,
+    executionRecord
+  );
+  t.teardown(async () => {
+    await executionPgModel.delete(t.context.knex, executionRecord);
+    await executionPgModel.delete(t.context.knex, parentExecutionRecord);
+    await collectionPgModel.delete(t.context.knex, collectionRecord);
+    await asyncOperationsPgModel.delete(t.context.knex, asyncRecord);
+  });
+
+  const response = await request(t.context.app)
+    .get(`/executions/${executionRecord.arn}`)
+    .set('Accept', 'application/json')
+    .set('Authorization', `Bearer ${jwtAuthToken}`)
+    .expect(200);
+
+  const executionResult = response.body;
+  const expectedRecord = await translatePostgresExecutionToApiExecution(
+    executionRecord,
+    t.context.knex
+  );
+
+  t.is(executionResult.arn, executionRecord.arn);
+  t.is(executionResult.asyncOperationId, asyncRecord.id);
+  t.is(executionResult.collectionId, `${collectionRecord.name}___${collectionRecord.version}`);
```

Comment:
Same comment as above. We have a helper for this if it makes sense to import here. `constructCollectionId`

Reply:
Agreed - it's always a pain to lookup, so I shorthand with the intent to go back and sort, and forgot it here.      Good catch.

Reply:
f13521635